bluesky/run_engine.py

```diff
@@ -78,8 +79,11 @@ def __set__(self, obj, value):
         old_value = self.__get__(obj)
         super().__set__(obj, value)
         value = self.__get__(obj)
-        obj.log.info("Change state on %r from %r -> %r",
-                     obj, old_value, value)
+        tags = {'old_status': old_value,
+                'direction': '--->>>',
```

Comment:
I see that this was inspired by 'direction' in caproto. I don't think we need it here, because it always changes from old to new.

Please name these tags `old_state` and `new_state`. The word "status" usually refers to ophyd Status objects, so this could be confusing.

Reply:
Without arrow, I feel [state1] [state2] looks unclear. Probably, [Old: <state1>] [New: <state2>]?

Reply:
I think in this case the log message makes it clear. Elsewhere I suggested that we do not need to include the extras in the Formatter the way we did with caproto.

Reply:
I think in this case the log message makes it clear. Elsewhere I suggested that we do not need to include the extras in the Formatter the way we did with caproto.

Reply:
Also, if we don't have that tag, it isn't very clear how Filter work. not clear which one is filtered out.

Reply:
I don't understand how having a tag `'direction'` that always has the same value is helpful.

Reply:
@ke-zhang-rd This is the sort of thing we are trying to avoid by having the design discussions on the issue before investing work in the code.